CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2.21.0 (2024-02-06)
+
+### Feat
+
+- Implement new currencies logic (#776)
+
```

Comment:
Should I add an entry here for the 3DS upgrade @semirp ?
I think it would be good to draw attention to it, even if there are no changes in this SDK

Reply:
yup @NQuinn27